libraries/standard/mqtt/src/iot_mqtt_api.c

```diff
@@ -1066,8 +1074,10 @@ IotMqttError_t IotMqtt_Connect( const IotMqttNetworkInfo_t * pNetworkInfo,
 {
     IOT_FUNCTION_ENTRY( IotMqttError_t, IOT_MQTT_SUCCESS );
     bool ownNetworkConnection = false;
-    IotNetworkError_t networkStatus = IOT_NETWORK_SUCCESS;
-    IotTaskPoolError_t taskPoolStatus = IOT_TASKPOOL_SUCCESS;
+    /* Initialize to failure status */
+    IotNetworkError_t networkStatus = IOT_NETWORK_FAILURE;
```

Comment:
What was the reason for changing these initial values?

I realize that in most cases the initial value of these variables are unused.  The only reason I initialized them is so that they are not garbage values when debugging.

Reply:
See the internal review CR for context/comments.